api/app/tests/c_haines/test_c_haines_worker.py

```diff
@@ -63,7 +33,34 @@ def mock_requests_get(*args, **kwargs):
     monkeypatch.setattr(requests, 'get', mock_requests_get)
 
 
-@pytest.mark.usefixtures('mock_download', 'mock_get_c_haines_model_run', 'mock_get_prediction_model')
+@pytest.fixture()
+def mock_minio(monkeypatch):
+    """ mock minio """
+    def mock_get_minio_client(*args, **kwargs):
+        """ mock get client """
+        class MockMinio(DefaultMockMinio):
+            """ Mock minio object """
+
+            def list_objects(self, bucket_name, prefix=None, recursive=False,
+                             start_after=None, include_user_meta=False,
+                             include_version=False, use_api_v1=False) -> Iterator[Object]:
+                """ mock list objects.
+                """
+                if prefix in ('c-haines-polygons/kml/GDPS/2020/5/21/0/2020-05-21T00:00:00.kml',
+                              'c-haines-polygons/json/GDPS/2021/5/21/0/2021-05-21T00:00:00.json'):
+                    # The worker checks if the objects already exist - we want to of those checks
```

Comment:
confused by this comment - we want to do what?

Reply:
I've tried to make it a bit more clear (we want it to not find these two files, but find all others).

Reply:
ohhhhh wait I just got struck with a bolt of clarity - see suggested change